x-pack/plugins/apm/public/context/url_params_context/url_params_context.test.tsx

```diff
@@ -81,8 +80,8 @@ describe('UrlParamsContext', () => {
     // force an update
     wrapper.setProps({ abc: 123 });
     const params = getDataFromOutput(wrapper);
-    expect(params.start).toEqual(getParsedDate('now-1d/d'));
-    expect(params.end).toEqual(getParsedDate('now-1d/d', { roundUp: true }));
+    expect(new Date(params.start).getTime()).not.toBeNaN();
+    expect(new Date(params.end).getTime()).not.toBeNaN();
```

Comment:
maybe add a snapshot test here? Or a literal value is also fine. But it might be useful to at least see the value of params here.

Reply:
The problem is that it's using a range relative to "now". I removed the export of `getParsedDate` because it only was used in this test. We're testing that it's parses relative time ranges, so essentially I'm testing if it's returning valid dates here and the specific times are not relevant. I'll see what else I can come up with.